src/generate-packages.ts

```diff
@@ -39,7 +39,7 @@ export default async function generatePackages(dt: FS, allPackages: AllPackages,
         if (tgz) {
             await writeTgz(pkg.outputDirectory, `${pkg.outputDirectory}.tgz`);
         }
-        log(` * ${pkg.libraryName}`);
+        log(` * ${pkg.desc}`);
```

Comment:
Adds the version of the package that’s being generated, rather than simply repeating the package name multiple times.